lib/install/config/loaders/installers/elm.js

```diff
@@ -4,11 +4,11 @@ const { env } = require('../configuration.js')
 const elmSource = path.resolve(process.cwd())
 
 const loaderOptions = () => {
-    if (env.NODE_ENV === "production") {
-        return `elm-webpack-loader?cwd=${elmSource}`
-    }
+  if (env.NODE_ENV === 'production') {
```

Comment:
:art: Thanks for doing this! Would you want to check for *development* and *test* environments instead? This way it'd be more of a *whitelist*. Those environments in the whitelist would get elm-hot-loader, debug, etc. support while the rest would get a more production-like setup. I mention this because I've worked in environment where you might have the following production-like environments:

- review
- stage
- production

By limiting the additional options to only development and test environments this might allow others to add additional environments, if need be, without having to modify the code further. Might make for a nice experience.